docs/content/latest/explore/query-1-performance/pg-stat-statements.md

```diff
@@ -0,0 +1,175 @@
+ calls               | bigint           |           |          |         | plain    | 
+ total_time          | double precision |           |          |         | plain    | 
+ min_time            | double precision |           |          |         | plain    | 
+ max_time            | double precision |           |          |         | plain    | 
+ mean_time           | double precision |           |          |         | plain    | 
+ stddev_time         | double precision |           |          |         | plain    | 
+ rows                | bigint           |           |          |         | plain    | 
+ shared_blks_hit     | bigint           |           |          |         | plain    | 
+ shared_blks_read    | bigint           |           |          |         | plain    | 
+ shared_blks_dirtied | bigint           |           |          |         | plain    | 
+ shared_blks_written | bigint           |           |          |         | plain    | 
+ local_blks_hit      | bigint           |           |          |         | plain    | 
+ local_blks_read     | bigint           |           |          |         | plain    | 
+ local_blks_dirtied  | bigint           |           |          |         | plain    | 
+ local_blks_written  | bigint           |           |          |         | plain    | 
+ temp_blks_read      | bigint           |           |          |         | plain    | 
+ temp_blks_written   | bigint           |           |          |         | plain    | 
+ blk_read_time       | double precision |           |          |         | plain    | 
+ blk_write_time      | double precision |           |          |         | plain    | 
+View definition:
+ SELECT pg_stat_statements.userid,
+    pg_stat_statements.dbid,
+    pg_stat_statements.queryid,
+    pg_stat_statements.query,
+    pg_stat_statements.calls,
+    pg_stat_statements.total_time,
+    pg_stat_statements.min_time,
+    pg_stat_statements.max_time,
+    pg_stat_statements.mean_time,
+    pg_stat_statements.stddev_time,
+    pg_stat_statements.rows,
+    pg_stat_statements.shared_blks_hit,
+    pg_stat_statements.shared_blks_read,
+    pg_stat_statements.shared_blks_dirtied,
+    pg_stat_statements.shared_blks_written,
+    pg_stat_statements.local_blks_hit,
+    pg_stat_statements.local_blks_read,
+    pg_stat_statements.local_blks_dirtied,
+    pg_stat_statements.local_blks_written,
+    pg_stat_statements.temp_blks_read,
+    pg_stat_statements.temp_blks_written,
+    pg_stat_statements.blk_read_time,
+    pg_stat_statements.blk_write_time
+   FROM pg_stat_statements(true) pg_stat_statements(userid, dbid, queryid, query, calls, total_time, min_time, max_time, mean_time, stddev_time, rows, shared_blks_hit, shared_blks_read, shared_blks_dirtied, shared_blks_written, local_blks_hit, local_blks_read, local_blks_dirtied, local_blks_written, temp_blks_read, temp_blks_written, blk_read_time, blk_write_time);
+```
+
```

Comment:
All examples: please provide sample output, if you can.

Reply:
> All examples: please provide sample output, if you can.

This output is related to the user's query, I suggest not increasing the output, but letting the user do the query in their own database.

Reply:
If you look at the rest of our docs, you'll notice that we provide sample output with (almost) every command or query...

Reply:
> If you look at the rest of our docs, you'll notice that we provide sample output with (almost) every command or query...

OK. I will add some output.